source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0005
+
+<b>Functional Description</b>
+
+The maximum value is 1999 as defined in the XSD pattern.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0006
+
+<b>Functional Description</b>
+
+&lt;TRANSPORT EQUIPMENT-GOODS REFERENCE.Declaration goods item number&gt; is filled in with the item number of the goods concerned as provided in Declaration goods item number.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0007
+
+<b>Functional Description</b>
+
+The Header shall be included if the field &lt;TRANSIT OPERATION.MRN&gt; is PRESENT in the rejected message, and can be included in the CD917C.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0009
+
+<b>Functional Description</b>
+
+IF &lt;FUNCTIONAL ERROR.Error code&gt; is in SET {90, 93} <br/>THEN &lt;FUNCTIONAL ERROR.Error pointer&gt; shall include the &lt;TRANSIT OPERATION.MRN&gt; of the rejected message <br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is in SET {92, 51, 52}<br/>THEN &lt;FUNCTIONAL ERROR.Error pointer&gt; shall include the &lt;Root Element&gt;<br/>ELSE &lt;FUNCTIONAL ERROR.Error pointer&gt; shall include the XPath location to point to the Data Item or the Data Group that caused the error.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0010
+
+<b>Functional Description</b>
+
+IF &lt;FUNCTIONAL ERROR.Error code&gt; is EQUAL to '12'<br/>THEN &lt;FUNCTIONAL ERROR.Error reason&gt; shall point to the Codelist number against which validation failed (ie CLxxx)<br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is in SET {13, 15} <br/>THEN &lt;FUNCTIONAL ERROR.Error reason&gt; shall point to the Condition/Technical Rule number against which validation failed (ie Cxxxx or Txxxx),<br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is EQUAL to '14' <br/>THEN &lt;FUNCTIONAL ERROR.Error reason&gt; shall point to the Rules/Technical Rule number against which validation failed (ie Rxxxx or Txxxx)<br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is EQUAL to '50' <br/>THEN &lt;FUNCTIONAL ERROR.Error reason&gt; shall point to the Transitional Constraint number against which validation failed (ie Exxxx or Bxxxx),<br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is in SET {51, 52} <br/>THEN the &lt;FUNCTIONAL ERROR.Error reason&gt; shall be:<br/>-  'ieCAvB' if exception is thrown by ieCA<br/>-  'NCAvB' if exception is thrown by NTA/NECA,<br/>ELSE the &lt;FUNCTIONAL ERROR.Error reason&gt; shall have the value 'N/A'
+
+<b>Technical Description</b>
+
+N/A
+
+##G0011
+
+<b>Functional Description</b>
+
+It contains the text of the error returned by the XML parser or XML validator.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0012
+
+<b>Functional Description</b>
+
+It should include the XPath location of the error. If the XPath string is to be truncated (i.e. if the length of the string is greater than 512 characters long), then the data item should not be used.
+
```

Comment:
Grammar/style issues -
- See earlier comment in this PR review about use of Latinate words and abbreviations. Suggest changing `i.e.` to `that is,` (note comma).
- Suggest removing "then" because it is a redundant word (the conditionality of the sentence is implied by the comma).